pallets/liquid-staking-v2/src/types.rs

```diff
@@ -34,10 +20,60 @@ pub struct UnstakeMisc<Balance> {
 
 /// The matching pool's total stake & unstake amount in one era
 #[derive(Copy, Clone, Eq, PartialEq, Default, Encode, Decode, RuntimeDebug)]
-pub struct MatchingLedger<Balance> {
+pub struct MatchingLedger {
```

Comment:
Is it necessary?

Reply:
without this I may need to add many restrictions for MathingLedger to make it supports `-` `+` operator, I think these restrictions are not necessary for now